/** @file
	@brief Header

	@date 2012

	@author
	Ryan Pavlik
	<rpavlik@iastate.edu> and <abiryan@ryand.net>
	http://academic.cleardefinition.com/
	Iowa State University Virtual Reality Applications Center
	Human-Computer Interaction Graduate Program
	
*/

//           Copyright Iowa State University 2012.
//  Distributed under the Boost Software License, Version 1.0.
//     (See accompanying file LICENSE_1_0.txt or copy at
//           http://www.boost.org/LICENSE_1_0.txt)

#pragma once
#ifndef INCLUDED_mainwindow_h_GUID_49C47E36_F14E_4A98_6059_2A4E40DF79C3
#define INCLUDED_mainwindow_h_GUID_49C47E36_F14E_4A98_6059_2A4E40DF79C3


// Internal Includes
// - none

// Library/third-party includes
// - none

// Standard includes
// - none

#endif // INCLUDED_mainwindow_h_GUID_49C47E36_F14E_4A98_6059_2A4E40DF79C3

